src/documentmanager/documentmanager.cpp

```diff
@@ -325,43 +333,39 @@ QMdiArea* DocumentManager::lastAreaInFocus()
     // search for area which accomodates last doc in focus
     for (const auto& area: mDocAreas)
     {
-        auto subWdwList = area->subWindowList();
-
-        areaInFocusIter = std::find_if(subWdwList.cbegin(), subWdwList.cend(),
-                                       [this](const auto& doc)
-        {
-            return static_cast<CodeEditor*>(doc->widget()) == mpPrevEditorInFocus;
-        });
+        auto currentWdw = area->currentSubWindow();
```

Comment:
auto **currentWdw** = area->currentSubWindow();
I suppose currentWindow is better name for variable

Reply:
fixed